src/client.rs

```diff
@@ -32,16 +32,23 @@ pub struct Client {
 
 impl Client {
     /// Constructs a new `Client`.
+    #[cfg(feature = "default-tls")]
     pub fn new() -> ::Result<Client> {
         let mut client = try!(new_hyper_client());
         client.set_redirect_policy(::hyper::client::RedirectPolicy::FollowNone);
-        Ok(Client {
+        Ok(Client::with_hyper_client(client))
+    }
+
+    /// Constructs a new `Client` with the provided `::hyper::Client` without
+    /// altering any settings.
```

Comment:
I don't know to be honest. I was considering a note like "if you use this method, you should look at the source of reqwest to create a client that's configured the same way as reqwest is expecting".

My main question is whether someone will someday want to use the hyper redirect policy rather than the reqwest one - what happens then? Or some other method on hyper that reqwest wants to set a default for (e.g. reqwest wants to start configuring timeouts and wants to set a different default value to hyper)?

How about I that "this method will configure your hyper::Client to use the reqwest default settings", and then if someone doesn't want that then they can make a `with_hyper_client_raw` method or similar?